DOCUMENTATION.md

```diff
@@ -14,6 +14,17 @@ This section describes all the possible options you can put in a service.toml fi
 You should create one different service.toml for each command you want to run. 
 A part from the `user` parameter, everything should work even with an unprivileged user.
 
+### Service templating
+Services can, but not have to, be templated. Currently templating works only via environment variables. Templating engine uses similar mechanism to popular jinja2 engine.
```

Comment:
I guess this should be also updated.

Reply:
Not sure if it would be worth it out a link to the crate's doc, in order to allow users to understand which features are supported... WDYT?

Reply:
Should be okay, 
I also forgot about one of the files, I noticed during manual testing.
Please resolve the comment if you are satisfied with the result :)